### PR TITLE
My first commit

### PR DESCRIPTION
**Pull request by kennylauph**
I am not a git virgin anymore

### CODE REVIEW

File: lesson6.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      Hi there everyone. I am Kenny,
+      I Love Disney and welcome to my page.
+      I am sure you can find something magical here.
+      </h5>
+            
+      <h5>
+      Feel free to support me and CLS - Commnet,  Like & Share.
+      </h5>
+
+    </div>
```
Comment (kongkers) [inline lesson6.html:121]: tabs would be good here...

File: lesson6.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      <h2>Hi, I am Kenny!</h2>
+      
+      <h5>
+      Hi there everyone. I am Kenny,
+      I Love Disney and welcome to my page.
+      I am sure you can find something magical here.
+      </h5>
+            
+      <h5>
+      Feel free to support me and CLS - Commnet,  Like & Share.
```
Comment (kongkers) [inline lesson6.html]: spelling mistake here

File: lesson6.html (hunk 1 above)
Comment (kennylauph) [reply lesson6.html:121]: dont care

File: lesson6.html (hunk 2 above)
Comment (kennylauph) [reply lesson6.html]: fixed

**Pull request merged**